synapse/module_api/__init__.py

```diff
@@ -1452,6 +1452,57 @@ async def get_monthly_active_users_by_service(
             start_timestamp, end_timestamp
         )
 
+    async def create_room(
+        self,
+        user_id: str,
+        config: JsonDict,
+        ratelimit: bool = True,
+        creator_join_profile: Optional[JsonDict] = None,
+    ) -> Tuple[str, Optional[str]]:
+        """Creates a new room.
+
+        Added in Synapse v1.66.0.
```

Comment:
```suggestion
        Added in Synapse v1.65.0.
```

Again, my bad, sorry I got confused :sweat_smile:

Reply:
No problem, fixed here: 7618a8d